src/components/TextInput/baseTextInputPropTypes.js

```diff
@@ -14,6 +14,9 @@ const propTypes = {
     /** Default value - used for non controlled inputs */
     defaultValue: PropTypes.string,
 
+    /** False when Value set by setNativeProps */
```

Comment:
This does not tell why this prop should be used at all. I think you should add a proper explanation and a warning to not use it unless needed. Give a reason why we have to introduce this prop. It is fine if the comment extends multiple lines.

Reply:
Thanks for taking a look at it Rajat